src/components/CustomDropdown/style.css

```diff
@@ -0,0 +1,48 @@
+.menu {
+    width: var(--dropdown-menu-width);
+    background-color: var(--dark-theme-dropdown-menu-bg);
+    border-radius: 3px;
+    padding: 8px;
+}
+
+.menu.narrow {
+    width: calc(var(--dropdown-menu-width) - 50px);
+}
```

Comment:
This file replaces the now deleted stylesheets for `HelpMenu` and `LoadFileMenu`.

This was the only style difference between the two to manage for now.